tests/test_keyboard_shortcuts_route.py

```diff
@@ -0,0 +1,9 @@
+OK_STATUS_CODE = 200
```

Comment:
Great PR!
To make it even better, try adding more tests

Reply:
Thanks idan!
Working on it :)